### PR TITLE
Adds five new bundles to traitors. Removes random bundles.

### PR DESCRIPTION
**Pull request by KazeEspada**
Removes random bundles because they were always worse then the surplus crate.

Five bundles are now buyable.

**Dark Lord:**
-Two energy swords
-A TK injector
-A chamelon voice changing gas mask
-An agent ID
-Chaplain Robes

**Sleeping Carp**-A bit iffy on the balance of this bundle
-Sleeping Carp Scroll
-Throwing Star Kit(includes the two bolas)
-Bo staff
-Chameleon helmet, jumpsuit, and mask
-Agent Card
-Smoke Grenade

**Nuclear Operative**
-Blood Red Hardsuit
-C-20r
-Weapon Authetication Implant
-Explosive Implant
-Agent Card
-Syndicate Mask
-C4

**Implant Bundle**
-Freedom Implant
-Explosive Implant
-Adrenaline Implant
-EMP Implant
-Storage Implant

**Operator Bundle**
-Chameleon Jumpsuit
-Agent ID
-Stechkin
-Silencer
-Briefcase Full of Dosh
-Body Bag
-Mulligan Syringe
-Syndie Smokes

:cl: KazeEspada
remove:Random Bundles are removed from the traitor uplink
add: Adds five new bundles for 20 TC. Darklord, Operative, Sleeping Carp, Operator, and Implant
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
:+1:  to the removal of random bundles, it allows for more interesting shit to go down.

**Comment by KorPhaeron** [comment]
>punishing players for choosing their own gear/plan/playstyle by making preconfigured sets of gear way cheaper

Sounds pretty dumb

**Comment by Zilenan91** [comment]
You can't really make a playstyle with syndi gear though. Almost 100% of the traitor arsenal is just stuff that allows you to kill more efficiently or have slightly more utility.

**Comment by KazeEspada** [comment]
These sets are viable but not the best you could get

**Comment by KazeEspada** [comment]
None contain emags

**Comment by PKPenguin321** [comment]
Pretty neat but RIP implant bundle being rare and having an uplink implamt

**Comment by KazeEspada** [comment]
You could just buy an uplink implant

**Comment by PKPenguin321** [comment]
Not the same

**Comment by KazeEspada** [comment]
However, in that same strain you could also get a shit bundle.

**Comment by Lumipharon** [comment]
Lucky it was *optional*
I don't see any reason to remove optional things, just because memecrates have like, FIFTY TC worth of shit in them.

**Comment by KazeEspada** [comment]
Its mostly a trap but i could easily enable it.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -104,6 +105,61 @@
...
+
+
+/obj/item/weapon/storage/box/syndie_kit/implantbundle/New() //Makes you slippery like a fish. Worse case scenario, just *deathgasp
+	..()
+	new /obj/item/weapon/implanter/freedom(src)
+	//new /obj/item/weapon/implanter/uplink(src)
+	new /obj/item/weapon/implanter/emp(src)
+	new /obj/item/weapon/implanter/adrenalin(src)
+	new /obj/item/weapon/implanter/explosive(src)
+	new /obj/item/weapon/implanter/storage(src)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: This one isn't particularly useful....at all.

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Security will never catch you, and if they do, you can get away. And if they kill you, they still get fucked.

Comment (Zilenan91) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Yeah I blew up the Warden once with this bundle when they arrested me for spamming a bunch of carp from the uplink implant it gave me.

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Uplink implant was removed because now it's a buyable bundle rather than random.

Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Yeah, that's the point I was making ;with the uplink, it's a really good bundle. Without it, it's pretty mediocre.

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: There are tons of new implants or even cyber implants, give me some ideas.

**Comment by Shadowlight213** [comment]
Is it still possible to get the toy AI module or no? iirc that one was only possible to get via the ai subversion bundle.

**Comment by KazeEspada** [comment]
For now, no. It should honestly be a 1tc traitor toy.

**Comment by Lumipharon** [comment]
it actually has a really high illegal origin tech so it's useful for R&D.

**Comment by Steelpoint** [comment]
>Nuclear Operative bundle

Can we PLEASE keep Nuke Ops to their own game mode?

**Comment by CPTANT** [comment]
Bundles punish determining your own effective gear.

**Comment by KazeEspada** [comment]
@Steelpoint The entire point of the operative bundle is metabreaking.

@CPTANT None of the bundles contain the EMAG and a lot of them are filled with bloat items(like the syndicate mask or the chameleon items).

**Comment by Anonmare** [comment]
Operator bundle seems a little weak, I'd say replace the !!DOSH!! with no-slip chameleon shoes to keep with the tacticool theme. 

I also have doubts regarding the C-20 being in the Nuke bundle. Make it a donksoft version with spare riot ammo, otherwise it'll be hilariously OP.

**Comment by KazeEspada** [comment]
The dosh stays, but I could add some no-slips.

I had some doubts about the C-20 with the nuke bundle, but I felt that the red hardsuit alone wasn't enough to convince the crew. I will gather more feedback on the C-20.

**Comment by CPTANT** [comment]
The c-20 only has one mag right? If that is the case it isnt really that great.

**Comment by Zilenan91** [comment]
You can print more ammo for it in a hacked autolathe, .45 ammo, but yeah no spare mags so you have to reload the ghetto way by taking out the magazine and then clicking on it with the ammo box.

**Comment by Steelpoint** [comment]
>metabreaking

While we're at it why don't we fucking let traitors dress themselves up as Wizards and give them some weak spells, or let them look like a Shadowling with the Sling Phase ability.

Hell let them buy their way into being a blob, now NO ONE will ever know what the round is.

**Comment by KazeEspada** [comment]
I thought about the wizard idea. Don't you tempt me.
Also, Gorlex are apart of the Syndicate Coalition, and as far as we know, Wizards, Slings, and Blobs are not.

**Comment by PKPenguin321** [comment]
>While we're at it why don't we fucking let traitors dress themselves up as Wizards and give them some weak spells, or let them look like a Shadowling with the Sling Phase ability.

i'm not sure if you could strawman harder if you tried lmao

i wouldn't be opposed to a 1 TC wizard gimmick outfit for traitors actually

**Comment by KazeEspada** [comment]
Actually I was thinking 20TC for a 1 use Die of Fate.

**Comment by Zilenan91** [comment]
Do it

**Comment by KorPhaeron** [comment]
Yeah it sure would be a lot of fun having 8 traitors all try and roll for wizard and then have the round be pseudo extended when they failed

**Comment by Fox-McCloud** [comment]
20 TC item to make yourself a changeling, as a traitor, when? =p

**Comment by KorPhaeron** [comment]
Sorry you wasted your time on this, but I told you not to do it before you started.

Also

Cheridan: then make the bundles better but selectable bundles are a bad idea
Cheridan: since yeah it's like "get a better value for your TC in exchange for your originality"

king of the coders agrees

**Pull request closed without merging**